internal/api/mfa.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/aaronarduino/goqrsvg"
 	svg "github.com/ajstarks/svgo"
 	"github.com/boombuler/barcode/qr"
+	"github.com/go-webauthn/webauthn/protocol"
```

Comment:
Maybe to aid readability?

```suggestion
	wbnprotocol "github.com/go-webauthn/webauthn/protocol"
```